ansible/roles/osdsdock/tasks/main.yml

```diff
@@ -15,24 +15,24 @@
 ---
 - name: include scenarios/lvm.yml
   include: scenarios/lvm.yml
-  when: deploy_project != "gelato" and enabled_backend == "lvm"
+  when: deploy_project != "gelato" and "lvm" in enabled_backends
 
 - name: include scenarios/ceph.yml
   include: scenarios/ceph.yml
-  when: deploy_project != "gelato" and enabled_backend == "ceph"
+  when: deploy_project != "gelato" and "ceph" in enabled_backends
 
 - name: include scenarios/cinder.yml
   include: scenarios/cinder.yml
   when:
-    - deploy_project != "gelato"
-    - enabled_backend == "cinder" and use_cinder_standalone == false
+   - deploy_project != "gelato" and "cinder" in enabled_backends
```

Comment:
Sorry I don't think it would work, because `enabled_backends` is also string type and you can't compare two strings using `in` matching rules.

Reply:
Hi @leonwanghui , I have done testing, it is working fine for me.

Reply:
![image](https://user-images.githubusercontent.com/45617130/53635850-661b0400-3c44-11e9-9b0d-ce4922dd5457.png)
![image](https://user-images.githubusercontent.com/45617130/53635856-6b784e80-3c44-11e9-87a6-ea92070ee58d.png)

Reply:
logs of osdsdock
![image](https://user-images.githubusercontent.com/45617130/53635915-995d9300-3c44-11e9-94d5-53f6d0c40b65.png)

Reply:
opensds.conf file
![image](https://user-images.githubusercontent.com/45617130/53636002-ca3dc800-3c44-11e9-83f8-ad805ba4ab4d.png)

Reply:
I have tested it, thanks!